packages/fxa-payments-server/src/components/SubscriptionForm/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { injectStripe, CardElement } from 'react-stripe-elements';
+
+class SubscriptionForm extends React.Component {
+  
+  render() {
+    const {
+      plansByProductId,
+    } = this.props;
+    
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <ul>
+          <li>
+            <p>Plans</p>
+            <ul>
+            {plansByProductId.length === 0 ? (
+              <li>No plans found.</li>
+            ) : (
+              plansByProductId.map(({ plan_id, product_id }) => (
+                <li key={plan_id}>
+                  <input type="radio" name="plan_id" value={plan_id} /> {plan_id}
+                </li>
+              ))
+            )}
+            </ul>
+          </li>
+          <li>
+            <p>Name</p>
+            <input name="name" />
+          </li>
+          <li>
+            <p>Card details (e.g. 4242 4242 4242 4242)</p>
+            <CardElement style={{base: {fontSize: '18px'}}} />
+          </li>
+          <li>
+            <p>Confirm</p>
+            <button>Confirm order</button>
+          </li>
+        </ul>
+      </form>
+    );
+  }
+
+  handleSubmit = (ev) => {
+    const {
+      createSubscription,
+    } = this.props;
+    
+    ev.preventDefault();
+    
+    const data = new FormData(ev.target);    
```

Comment:
👍 I've been using refs for this all this time, but this FormData pattern is much nicer.